scripts/copy_collection_files.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import json
+import os
+import shutil
+
+
+def copy_collection_file(dump_files, destination_prefix, working_dir):
+    file_log = open(os.path.join(working_dir, "files.log"), "w")
+
+    for dump_file in dump_files:
+        with open(os.path.join(working_dir, dump_file), "r") as json_dump:
+            data = json.load(json_dump)
+            for record in data:
+                legacy_record_files = record["files"]
+                for legacy_record_file in legacy_record_files:
+                    full_path = legacy_record_file["full_path"]
+                    # important: last slash
+                    path_to_replace = "/opt/cdsweb/var/data/files/"
+
+                    rel_path = full_path.replace(path_to_replace, "")
+                    destination_path = os.path.join(destination_prefix, rel_path)
+                    parent_dest_path = os.path.dirname(destination_path)
+                    if not os.path.exists(parent_dest_path):
+                        os.makedirs(parent_dest_path)
+                    shutil.copy(full_path, destination_path)
```

Comment:
discussed to use the eosclient and permission based tokens that should replace the copy command here and the need for `kinit`